### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by iakovleva**
Implements #17

### CODE REVIEW

File: pylab/website/static/img/logo.svg [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Creator: CorelDRAW -->
```
Comment (sirex) [inline pylab/website/static/img/logo.svg]: Try some time [Inkscape](https://inkscape.org) an open source alternative to CorelDRAW.

Comment (iakovleva) [inline pylab/website/static/img/logo.svg]: Thanks for the advice, I'll look.

**Comment by sirex** [comment]
Looks good to me. I'm merging it.

Also I added some changes, I hope you don't mind :)

First I saw, that images looks slightly different compared to svg file, svg file had white reflection and `pylab.lt` on top of that reflection. I removed `pylab.lt` since it is hardly visible on 45x50 logo image and also I didn't liked the font. Font looked like `Courier New`, I'm not use if we can legally use it since it is a font with [a non-free licence](http://www.microsoft.com/typography/fonts/font.aspx?FMID=1735).

And also, to make svg match png, regenerated all png images from svg.

Thanks for the logo!

**Comment by iakovleva** [comment]
Oh, I had 2 variants of logo, just made svg file of wrong one. I wasn't going to use that with pylab.lt lettering. Sorry for adding more work with it.

**Pull request merged**